### ISSUE TITLE
Goals: add new button

### ISSUE THREAD

**Issue by MollieCourtenay** [healthlocker/healthlocker#359]
At the height it is now. 
Aligned right, aligned to the end of the grey line.

**Comment by katbow** [comment]
@MollieCourtenay is this a question for @indiamindwave24 before we start anything?

**Comment by MollieCourtenay** [comment]
I'll leave with @indiamindwave24

**Comment by MollieCourtenay** [comment]
agreed with india - that this just need to be aligned right.

**Comment by MollieCourtenay** [comment]
@Cleop can you do the same for the add new coping strategies button? Sorry i thought I had added this!

+ [ ] right align add new coping strategy button

**Comment by Cleop** [comment]
@MollieCourtenay I'm going to open that as a separate issue and put please test on this one.

**Comment by indiamindwave24** [comment]
👍
